<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RustDaVinci</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons.qrc">
    <normaloff>:/icons/RustDaVinci-icon.ico</normaloff>:/icons/RustDaVinci-icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_image_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>Load Image from File or URL</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>         Load Image...        </string>
    </property>
    <property name="icon">
     <iconset resource="../resources/icons.qrc">
      <normaloff>:/icons/load_image_icon.png</normaloff>:/icons/load_image_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="identify_ctrl_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="toolTip">
     <string>Capture the painting control area manually or automatically</string>
    </property>
    <property name="text">
     <string> Capture Control Area...</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/icons.qrc">
      <normaloff>:/icons/select_area_icon.png</normaloff>:/icons/select_area_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="paint_image_PushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="toolTip">
     <string>Paint the Image</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>         Paint Image           </string>
    </property>
    <property name="icon">
     <iconset resource="../resources/icons.qrc">
      <normaloff>:/icons/paint_image_icon.png</normaloff>:/icons/paint_image_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>220</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="toolTip">
     <string>Show RustDaVinci Settings</string>
    </property>
    <property name="text">
     <string>            Settings              </string>
    </property>
    <property name="icon">
     <iconset resource="../resources/icons.qrc">
      <normaloff>:/icons/settings_icon.png</normaloff>:/icons/settings_icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>220</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="preview_PushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Show Original Image and Preview of the Quantized Images</string>
    </property>
    <property name="text">
     <string>Show Image &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_ProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_TextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
